Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmon.yaml

```diff
@@ -1,7 +1,7 @@
 Parser:
   Title: Process Terminate Event ASIM parser for Microsoft Windows Security Events
-  Version: '0.2.1'
-  LastUpdated: Feb 23, 2022
+  Version: '0.3.1'
+  LastUpdated: June 3, 2024
```

Comment:
"LastUpdated" property exists but is not correct format. The expected format is, for example, "Jun 29, 2024"

Reply:
done